examples/passage_ranking.py

```diff
@@ -0,0 +1,141 @@
+# fmt: off
+import logging
+from pathlib import Path
+
+from farm.data_handler.data_silo import DataSilo
+from farm.data_handler.processor import RegressionProcessor, TextPairClassificationProcessor
+from farm.experiment import initialize_optimizer
+from farm.infer import Inferencer
+from farm.modeling.adaptive_model import AdaptiveModel
+from farm.modeling.language_model import LanguageModel
+from farm.modeling.prediction_head import RegressionHead, TextClassificationHead
+from farm.modeling.tokenization import Tokenizer
+from farm.train import Trainer
+from farm.utils import set_all_seeds, MLFlowLogger, initialize_device_settings, reformat_msmarco_train, reformat_msmarco_dev, write_msmarco_results
+from farm.evaluation.msmarco_passage_farm import msmarco_evaluation
+
+
+def text_pair_classification():
+    logging.basicConfig(
+        format="%(asctime)s - %(levelname)s - %(name)s -   %(message)s",
+        datefmt="%m/%d/%Y %H:%M:%S",
+        level=logging.INFO)
+
+    ml_logger = MLFlowLogger(tracking_uri="https://public-mlflow.deepset.ai/")
+    ml_logger.init_experiment(experiment_name="Public_FARM", run_name="Run_text_pair_classification")
+
+    ##########################
+    ########## Settings
+    ##########################
+    set_all_seeds(seed=42)
+    device, n_gpu = initialize_device_settings(use_cuda=True)
+    n_epochs = 2
+    batch_size = 64
+    evaluate_every = 500
+    lang_model = "bert-base-cased"
+    label_list = ["0", "1"]
+    train_filename = "train.tsv"
+    dev_filename = "dev.tsv"
+    generate_data = True
+    data_dir = Path("../data/msmarco_passage")
+    predictions_raw_filename = "predictions_raw.txt"
+    predictions_filename = "predictions.txt"
+    train_source_filename = "triples.train.small.tsv"
+    qrels_filename = "qrels.dev.tsv"
+    queries_filename = "queries.dev.tsv"
+    passages_filename = "collection.tsv"
+    top1000_filename = "top1000.dev"
+
+
+    # 0. Preprocess and save MSMarco data in a format that can be ingested by FARM models. Only needs to be done once!
+    if generate_data:
```

Comment:
Would it make sense to give here a quick example of the target format for the TextPairClassificationProcessor? (e.g text_a      text_b     label). Might help users to use it for their own dataset (which is not msmarco)